@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -137,6 +137,38 @@ const MAX_CORNER_ANGLE = Math.PI / 8;
  */
 const INVALID_ARRAY_INDEX = -1;
 
+function maxValue(array: ArrayLike<number>): number {
```

Comment:
Given that the index buffer does not contain indices that are out of range: the highest index can also be determined with `posAttr.count` which should be a bit faster than looping over the index buffer.

Reply:
ok